modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspaceExtension.java

```diff
@@ -71,8 +88,25 @@ public WorkspaceExtension(Settings settings) {
 		_bundleTokenPasswordFile = _getProperty(
 			settings, "bundle.token.password.file",
 			_BUNDLE_TOKEN_PASSWORD_FILE);
+
+		String workspaceProductKey = _getProperty(
+			settings, "product", (String)null);
+
+		_productInfo = _getProductInfo(workspaceProductKey);
```

Comment:
Can you sort the method calls that assign these fields?  like the rest of them are sorted, thanks!